move/move-ibc/sources/light_client.move

```diff
@@ -21,7 +21,7 @@ module ibc::light_client {
                     consensus_state_bytes
                 );
             return (client_state, consensus_state)
-        } else if (string::bytes(&client_type) == &b"statelens") {
+        } else if (string::bytes(&client_type) == &b"state-lens/ics23/mpt") {
```

Comment:
extract these client type names to constants?

Reply:
already did